src/bci_build/package/apache_tomcat.py

```diff
@@ -52,52 +54,44 @@ def _get_sac_supported_until(
         if os_version.is_tumbleweed
         else f"sac-apache-tomcat-{tomcat_major}-java{jre_version}-image",
         pretty_name="Apache Tomcat",
+        package_list=[
+            tomcat_pkg := (
+                "tomcat"
+                if tomcat_major == _TOMCAT_VERSIONS[0]
+                else f"tomcat{tomcat_major}"
+            )
+        ]
+        + [f"java-{jre_version}-openjdk", f"java-{jre_version}-openjdk-headless"],
         os_version=os_version,
         is_latest=(
             (os_version in CAN_BE_LATEST_OS_VERSION)
             and tomcat_major == _TOMCAT_VERSIONS[-1]
             and jre_version == 22
             and os_version.is_tumbleweed
         ),
-        version=f"{tomcat_major}-jre{jre_version}",
+        version=(tomcat_pkg_version := get_pkg_version(tomcat_pkg, os_version)),
```

Comment:
I'm not opposed to using our version fetching mechanics, but it causes more busy work on our side instead of the buildservice handling this automatically. Do we have to prefetch the versions or can we rely on the replace_using_package_version service instead?

Reply:
True, there was a reason why I converted that to the package version fetching logic but it no longer applies. reverted that part.